client/bindings/java/iota-client-java/lib/build.gradle

```diff
@@ -0,0 +1,161 @@
+plugins {
+    // Apply the java-library plugin for API and implementation separation.
+    id 'java-library'
+    id 'c'
+    id 'maven-publish'
+    id 'signing'
+    id 'com.google.osdetector' version '1.7.1'
+}
+
+dependencies {
+    // Use JUnit Jupiter for testing.
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.0'
+    // Dependencies that are used internally, and not exposed to consumers on their own compile classpath.
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.9'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
+    implementation group: 'commons-codec', name: 'commons-codec', version: '1.15'
+}
+
+repositories {
+    mavenCentral()
+}
+
+java {
+    sourceCompatibility = "1.8"
+    targetCompatibility = "1.8"
+
+    withJavadocJar()
+    withSourcesJar()
+}
+
+javadoc {
+    if(JavaVersion.current().isJava9Compatible()) {
+        options.addBooleanOption('html5', true)
+    }
+}
+
+task buildRustRelease(type:Exec){
+    workingDir 'native'
+
+    if (!project.hasProperty('buildTarget')) {
+        commandLine 'cargo', 'build', '--release'
+    } else {
+        commandLine 'cargo', 'build', '--target=' + buildTarget, '--release'
+    }
+}
+
+archivesBaseName = 'iota-client-java'
+version = '1.0.0-rc.2'
```

Comment:
rc-2 here still, we should define version in top level build.gradle and use that

Reply:
I tried this as it would be much nicer but unfortunately I can not access it then here for some reason, any idea how to access it here in the publication?